src/atoms/TextArea/__tests__/TextArea.web.test.js

```diff
@@ -0,0 +1,283 @@
+import React from 'react';
+import { fireEvent } from '@testing-library/react';
+import { renderWithTheme } from '../../../_helpers/testing';
+import TextArea from '../index';
+
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
+afterAll(() => jest.restoreAllMocks());
+
+const SAMPLE_ID = 'sample-id';
+const SAMPLE_LABEL = 'Sample Label';
+
+describe('<TextArea />', () => {
+  describe('variant', () => {
+    it('renders outlined variant with default props', () => {
+      const { container } = renderWithTheme(<TextArea variant="outlined" />);
+      expect(container).toMatchSnapshot();
+    });
+
+    it('renders filled variant with default props', () => {
+      const { container } = renderWithTheme(<TextArea variant="filled" />);
+      expect(container).toMatchSnapshot();
+    });
+  });
+
+  describe('id', () => {
+    it('should not have id attribute if not provded as prop', () => {
```

Comment:
typo. will fix.

Reply:
Looks good. Fix the typo and make sure to address comments from text input PR test file changes here.